src/parser/mod.rs

```diff
@@ -4861,11 +4865,37 @@ impl<'a> Parser<'a> {
                     None
                 };
                 AlterColumnOperation::SetDataType { data_type, using }
+            } else if self.parse_keywords(&[Keyword::ADD, Keyword::GENERATED]) {
+                let generated_as = if self.parse_keyword(Keyword::ALWAYS) {
+                    Some(GeneratedAs::Always)
+                } else if self.parse_keywords(&[Keyword::BY, Keyword::DEFAULT]) {
+                    Some(GeneratedAs::ByDefault)
+                } else {
+                    None
+                };
+
+                let _ = self.expect_keywords(&[Keyword::AS, Keyword::IDENTITY]);
```

Comment:
Why is this ignoring the result? shouldn't it at least check and return the error, like:

```suggestion
                self.expect_keywords(&[Keyword::AS, Keyword::IDENTITY])?;
```